pkg/controller/enterprisesearch/name/name.go

```diff
@@ -27,3 +28,7 @@ func Deployment(entName string) string {
 func Config(entName string) string {
 	return EntNamer.Suffix(entName, configSuffix)
 }
+
+func Scripts(entName string) string {
```

Comment:
It looks like this is not called anywhere, correct?

Reply:
Right! Good catch.